### PR TITLE
Refactor out DefaultFormatter (#35)

### PR DESCRIPTION
**Pull request by lbschanno**
Refactor out usage of DefaultFormatter. It is an internal-only unstable
API that is not recommended for users, and as such is unsuitable to
include in examples.

### CODE REVIEW

File: src/main/java/org/apache/accumulo/examples/filedata/ChunkInputFormat.java [hunk 1]
```diff
@@ -53,26 +54,72 @@ public void initialize(InputSplit inSplit, TaskAttemptContext attempt) throws IO
       }
 
       @Override
-      public boolean nextKeyValue() throws IOException, InterruptedException {
+      public boolean nextKeyValue() throws IOException {
+        System.out.println("nextKeyValue called");
```
Comment (milleruntime) [inline src/main/java/org/apache/accumulo/examples/filedata/ChunkInputFormat.java]: Should use the logger here.

**Review by milleruntime — CHANGES_REQUESTED** [review]
@lbschanno thanks for the contribution!  I just have a couple things for your changes...

I think copying the code from DefaultFormatter is fine but the two new methods should be placed in a Util class to be reused across the examples.  Right now you have duplicate code, which we try to avoid.

**Comment by lbschanno** [comment]
@milleruntime No problem. Please take a look at the update for the requested changes and let me know if anything else needs to be modified.

**Comment by milleruntime** [comment]
Looks good!  

With recent changes to Accumulo Master migrating to Java 11, the example repo is broken.  I will work on getting it up to date and then merge your PR.

**Comment by lbschanno** [comment]
Thanks, much appreciated!

**Review by milleruntime — APPROVED** [review]

**Pull request merged**